lib/aasm/instance_base.rb

```diff
@@ -14,15 +14,13 @@ def current_state
 
     def current_state=(state)
       @instance.aasm_write_state_without_persistence(state, @name)
-      # @current_state = state
     end
 
     def enter_initial_state
       state_name = determine_state_name(@instance.class.aasm(@name).initial_state)
       state_object = state_object_for_name(state_name)
 
       state_object.fire_callbacks(:before_enter, @instance)
-      # state_object.fire_callbacks(:enter, @instance)
```

Comment:
It make sense to remove redundant callbacks, I think we should make this change in AASM 5.